compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/ParserTestUtils.java

```diff
@@ -348,13 +348,23 @@ private static SyntaxKind getNodeKind(String kind) {
             case "PERCENT_TOKEN":
                 return SyntaxKind.PERCENT_TOKEN;
             case "EQUAL_LT_TOKEN":
-                return SyntaxKind.EQUAL_LT_TOKEN;
+                return SyntaxKind.LT_EQUAL_TOKEN;
```

Comment:
case should be `case "LT_EQUAL_TOKEN"`

Reply:
I have added it below. so this case was removed.